src/components/Payment/Payment.tsx

```diff
@@ -42,22 +45,24 @@ const Payment:React.FC<IProps> = ({componentPayment, openComponentPayment}) => {
 
     const submitBtn = () => {
         openComponentPayment(!componentPayment)
-
+        openComponentReview(true)
+        openEditBtn({...editBtn, payment: true})
     }
 
     return (
         <div className="payment">
-            <h2 className="payment__number">3</h2>
+            <h2 style={{color: !componentPayment ? "#999" : "#000"}} className="payment__number">3</h2>
+            <div style={{backgroundColor: !componentPayment ? "#999" : "#000"}} className="dash-payment"></div>
```

Comment:
It more maintainable to use two classes instead of having conditional inline-style